yt_dlp/extractor/freetv.py

```diff
@@ -0,0 +1,176 @@
+                raise ExtractorError("Response doesn't contain episode list")
+
+            for episode in season_episodes:
+                video_id = str_or_none(episode.get('contentID'))
+                video_url = episode.get('streamURL')
+
+                if determine_ext(video_url) == 'm3u8':
+                    formats, subtitles = self._extract_m3u8_formats_and_subtitles(
+                        video_url, video_id, 'mp4', fatal=False)
+                    self._sort_formats(formats)
+
+                    entries.append({
+                        'id': video_id,
+                        'title': episode.get('fullTitle'),
+                        'description': episode.get('description'),
+                        'formats': formats,
+                        'subtitles': subtitles,
+                        'thumbnail': episode.get('thumbnail'),
+                        'series': series_title,
+                        'series_id': traverse_obj(episode, ('contentMeta', 'displayMeta', 'seriesID')),
+                        'season_id': traverse_obj(episode, ('contentMeta', 'displayMeta', 'seasonID')),
+                        'season_number': int_or_none(
+                            traverse_obj(episode, ('contentMeta', 'displayMeta', 'seasonNum'))),
+                        'episode_number': int_or_none(
+                            traverse_obj(episode, ('contentMeta', 'displayMeta', 'episodeNum'))),
+                    })
+                else:
+                    self.report_warning("Manifest was not found", video_id)
+
+        return self.playlist_result(entries, display_id, series_title, series_description)
+
+
+class FreeTvMoviesIE(FreeTvBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?freetv\.com/(?:peliculas)/(?:[^/]+/)*(?P<id>[^/]+)'
+    _TESTS = [{
+        'url': 'https://www.freetv.com/peliculas/atrapame-si-puedes/',
+        'md5': 'dc62d5abf0514726640077cd1591aa92',
+        'info_dict': {
+            'id': '428021',
+            'title': 'Atrápame Si Puedes',
+            'description': 'md5:ca63bc00898aeb2f64ec87c6d3a5b982',
+            'ext': 'mp4',
+        }
+    }, {
+        'url': 'https://www.freetv.com/peliculas/monstruoso/',
+        'md5': '509c15c68de41cb708d1f92d071f20aa',
+        'info_dict': {
+            'id': '377652',
+            'title': 'Monstruoso',
+            'description': 'md5:333fc19ee327b457b980e54a911ea4a3',
+            'ext': 'mp4',
+        }
+    }]
+
+    def _real_extract(self, url):
+        display_id = self._match_id(url)
+        webpage = self._download_webpage(url, display_id)
+
+        content_id = self._search_regex(
+            (r'class=["\'][^>]+postid-(?P<video_id>\d+)',
+             r'<link[^>]+freetv.com/\?p=(?P<video_id>\d+)'
+             r'<div[^>]+data-params=["\'][^>]+post_id=(?P<video_id>\d+)'),
+            webpage, 'video_id', group='video_id')
+
+        return self._extract_video(content_id)
+
+
+class FreeTvSeriesIE(FreeTvBaseIE):
```

Comment:
In case we want to seperate the IE into single episode/series later, naming this FreeTvIE would provide some future-proofing. You can set an IE_NAME explaining it's series
```suggestion
class FreeTvIE(FreeTvBaseIE):
    IE_NAME = 'freetv:series'
```